test/verify/check-networkmanager-settings

```diff
@@ -186,6 +186,44 @@ class TestNetworkingSettings(NetworkCase):
         b.click('#network-ip-settings-cancel')
         b.wait_not_present("#network-ip-settings-dialog")
 
+    def testIpHelper(self):
```

Comment:
The `TestNetworkingSettings` class is rather "expensive": It starts *two* new VMs for each new test case, which is disproportionate to the "harmless" things that happen in this test. This should either be merged into `testOtherSettings`, or be moved into a new `@nondestructive` test class.

It's probably the easiest way to move this test into test/verify/check-networkmanager-basic.

As this does not check external APIs, please add `TestNetworkingBasic.testIpHelper` (the name after the move) to test/browser/run-test.sh before the `TestNetworkingBasic.testNoService` line. This will avoid running this test in packit (we need to not make this take too long).

Reply:
ack